administrator/components/com_fields/src/Plugin/FieldsPlugin.php

```diff
@@ -282,6 +278,12 @@ public function onCustomFieldsPrepareDom($field, \DOMElement $parent, Form $form
         $params = clone $this->params;
         $params->merge($field->fieldparams);
 
+        $layout = $params->get('form_layout');
```

Comment:
```suggestion
        $layout = $field->params->get('form_layout');
```
The layout is not found in $params